index.yaml

```diff
@@ -13210,7 +13210,7 @@ entries:
       and Prometheus rules combined with documentation and scripts to provide easy
       to operate end-to-end Kubernetes cluster monitoring with Prometheus using the
       Prometheus Operator.
-    digest: 4ce289a2bc56327093b8c59a76994988d368f09120f6e347c8fcdac69e894e39
+    digest: 1d108e2e068fe7f95071d031ffe65fd8543537569e79cef9df4159ba809fa681
```

Comment:
This digest looks is not same as in release-v2.8 and dev-v2.8.1 4ce289a2bc56327093b8c59a76994988d368f09120f6e347c8fcdac69e894e39.
Please recheck.